esinstall/src/rollup-plugins/rollup-plugin-strip-source-mapping.ts

```diff
@@ -9,7 +9,7 @@ export function rollupPluginStripSourceMapping(): Plugin {
   return {
     name: 'snowpack:rollup-plugin-strip-source-mapping',
     transform: (code) => ({
-      code: code.replace(/\/\/+#\s*sourceMappingURL=.+$/gm, ''),
+      code: code.replace(/[^'"`]\/\/+#\s*sourceMappingURL=.+$/gm, ''),
```

Comment:
Ok so this doesn’t strip the source map if a string character is right before it?

This works for me. But I wonder if there’s a way to also change this so that it only strips it if it’s literally the last string in the file (which may or may not end on a newline)

Reply:
Ah, unfortunately, I've seen a few mid-file, usually when the package has been pre-bundled in some way. But, there's a chance that esinstall actually caused that, and each individual transform had only one `sourceMappingURL ` comment.

This is enough for now and I don't want a regression unnecessarily, but if there are still bugs after this then let's revisit.

Reply:
Ah I thought there was a reason for scanning mid-file. Bug found 🙃 : https://github.com/snowpackjs/snowpack/pull/1505